app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,9 @@
   </head>
 
   <body>
+    <% flash.each do |type, message| %>
```

Comment:
@CjMoore @meyerhoferc @valerietrudell do you think it makes sense for us to determine which types of flash messages to use in specific methods so we have continuity across controllers? Just an idea!